src/models/partial_tx.ts

```diff
@@ -596,7 +596,7 @@ export class PartialTxInputData {
    */
   addSignatures(serialized: string) {
     const arr = serialized.split('|');
-    if (arr.length < 3 || arr[0] != 'PartialTxInputData' || arr[1] !== this.hash) {
+    if (arr.length < 2 || arr[0] != 'PartialTxInputData' || arr[1] !== this.hash) {
```

Comment:
Don't we have the `PartialTxInputData` string, the hash and the signatures? So we need the array at least with length 3, right?

Reply:
There is the case of a participant with no inputs requesting his signatures.
The input data part would be empty so we can either not return the signatures to the user (stating that the input data is empty) or allow the "no signatures" case and it seemed best to just have a noop case when the user "signature" does not contain any input data.